02_repeatString/solution/repeatString-solution.spec.js

```diff
@@ -0,0 +1,39 @@
+const repeatString = require("./repeatString");
```

Comment:
If you want the solutions to be easily verifiable (see [below](https://github.com/TheOdinProject/javascript-exercises/pull/234/files#r1092794141), would it be better if each `foo-solution.spec.js` file require'd the corresponding `"./foo-solution"` rather than just `"./foo"`?

Reply:
Good catch! Didn't realize I hadn't pushed that to my branch, should be up to date now!